### ISSUE TITLE
Loading spinner on sessions list could use a bit of top margin

### ISSUE THREAD

**Issue by ktagbnaw** [vector-im/riot-web#13238]
![riot](https://user-images.githubusercontent.com/59598719/79570072-1225e580-80b9-11ea-965d-bbca7b590453.png)

### Description

The loading spinner seems to be a bit too close to the "Sessions" description. A bit of top margin would probably fix this.

### Steps to reproduce

- Open "Security & Privacy" tab in settings.

### Version information

- **Platform**:  web

For the web app:

- **Browser**: Firefox 77.0a1
- **OS**: Windows
- **URL**: riot.im/develop (Commit 8122c62f24c418af3680510bf1c93a3dd3cf42b0)

**Comment by dbkr** [comment]
This should be a simple issue if someone new to the codebase would like to pick it up: basically a CSS tweak in matrix-react-sdk in the appropriate dialog.

**Comment by yuktea** [comment]
I would like to try the issue out, could you please assign it to me?

**Comment by SimonBrandner** [comment]
Yes, sure, if you have any questions/need help, be sure to visit https://matrix.to/#/#element-dev:matrix.org!
